front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/MostReactedIdeasWidget/messages.ts

```diff
@@ -2,16 +2,16 @@ import { defineMessages } from 'react-intl';
 
 export default defineMessages({
   mostReactedIdeas: {
-    id: 'app.containers.admin.ReportBuilder.MostReactedIdeasWidget.mostReactedIdeas',
-    defaultMessage: 'Most reacted ideas',
+    id: 'app.containers.admin.ReportBuilder.MostReactedIdeasWidget.mostReactedIdeas1',
```

Comment:
I think that as long as we keep referring to inputs as ideas in the code, we will keep making these copy mistakes... it's really confusing

Reply:
I agree. It's also not 100% clear to me whether we could use the word "Idea" in copies or we should rename it everywhere https://www.notion.so/I3-Nail-the-admin-user-journey-nudges-4bb6d91027f74f1bb02623c3d10146bc?d=184ee292369144deaf47a541567a517e&pvs=4#8046fa7e9a0848bf8fcaf8d04e295c3d